.github/workflows/e2e.yaml

```diff
@@ -118,3 +118,19 @@ jobs:
         run: export E2E_VERSION=jdk8-1.3 && bash -x test/e2e/run.sh e2e-profile/e2e-profile-test-runner --storage=elasticsearch
       - name: Profile Tests ES7(JDK8)
         run: export E2E_VERSION=jdk8-1.3 DIST_PACKAGE=apache-skywalking-apm-bin-es7.tar.gz ES_VERSION=7.4.2 && bash -x test/e2e/run.sh e2e-profile/e2e-profile-test-runner --storage=elasticsearch
+        
+  Browser:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+        with:
+          submodules: true
+      - name: Set environment
+        run: export MAVEN_OPTS='-Dmaven.repo.local=~/.m2/repository -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -XX:+CMSClassUnloadingEnabled -XX:+UseConcMarkSweepGC -XX:-UseGCOverheadLimit -Xmx3g'
+      - name: Compile & Install Test Codes
+        run: |
+          ./mvnw checkstyle:check apache-rat:check
+          ./mvnw -Dcheckstyle.skip -Drat.skip -T2 -Dmaven.compile.fork -Dmaven.compiler.maxmem=3072 -DskipTests clean install
+          ./mvnw -f test/e2e/pom.xml -pl e2e-base clean install
+      - name: Browser Tests ES6(JDK8)
+        run: export E2E_VERSION=jdk8-1.3 && bash -x test/e2e/run.sh e2e-browser/e2e-browser-es
```

Comment:
Please add the tests for different storage. And browser e2e tests should be in an separated `e2e-browser-perf.yaml` file.

Reply:
done